app/models/flood_risk_engine/enrollment.rb

```diff
@@ -42,6 +42,10 @@ def state_machine
       to: :state_machine
     )
 
+    def state
+      self.step
+    end
+
```

Comment:
alternative `alias_attribute :state, :step` - but if we start mixing up state and step (we will also have a status eg active, deregistered etc) then the code could be become harder to follow